typedoc.js

```diff
@@ -1,25 +1,8 @@
 module.exports = {
-  exclude: ['**/*spec.ts', './src/index.ts', './src/wasm/wasm_exec.js'],
+  exclude: ['**/testSetup/*', '**/*.spec.ts', './src/wasm/wasm_exec.js'],
   listInvalidSymbolLinks: true,
-  stripInternal: true,
+  excludeExternals: true,
   excludeNotExported: true,
   excludePrivate: true,
-  hideGenerator: true,
```

Comment:
Why remove hiding the generator?

Reply:
why would we hide it? I'm always a bit annoyed if people hide this kind of info. If I see the documentation of an open source project, I would like to know which tools they use. I don't see a point in hiding this.

Reply:
We are hiding it in the SDK but I agree with you.